client/src/components/Homepage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import LargePhoto from './LargePhoto.jsx';
+import SmallPhotos from './SmallPhotos.jsx';
+
+
+const Homepage = (props) => {
+  //create an array of 4 photos to create smaller photos
+  const photoArray = [];
```

Comment:
This feels redundant. Is props.photos already an array? Might be a good opportunity to use destructuring {photos}